11-payment-encoding.md

```diff
@@ -145,12 +162,12 @@ A writer MAY include one or more `f` fields. For bitcoin payments, a writer MUST
 `f` field to a valid witness version and program, or `17` followed by
 a public key hash, or `18` followed by a script hash.
 
-A writer MUST include at least one `r` field if it does not have a
-public channel associated with its public key.  The `r` field MUST contain
+A writer MUST include at least one `r` field if there is not a
+public channel associated with its public key. The `r` field MUST contain
 one or more ordered entries, indicating the forward route from a
-public node to the final destination.  For each entry, the `pubkey` is the
-node ID of the start of the channel, `short_channel_id` is the short channel ID
-field to identify the channel, `fee_base_msat`, `fee_proportional_millionths` and `cltv_expiry_delta` are as specified in [BOLT #7](07-routing-gossip.md#the-channel_update-message).  A writer MAY include more than one `r` field to
+public node to the final destination. For each entry, the `pubkey` is the
+node ID of the start of the channel; `short_channel_id` is the short channel ID
+field to identify the channel; and `fee_base_msat`, `fee_proportional_millionths` and `cltv_expiry_delta` are as specified in [BOLT #7](07-routing-gossip.md#the-channel_update-message). A writer MAY include more than one `r` field to
```

Comment:
...`fee_proportional_millionths`, and...

Reply:
Done.